src/js/components/validateData/validateValues/ValidateValuesFileComponent.jsx

```diff
@@ -27,7 +28,6 @@ const propTypes = {
 export default class ValidateDataFileComponent extends React.Component {
     constructor(props) {
         super(props);
-
```

Comment:
Might as well leave this space in for readability (not a big deal but since I've already asked for a different change, might as well update this while you're in there)